x-pack/qa/full-cluster-restart/src/test/java/org/elasticsearch/xpack/restart/MlConfigIndexMappingsFullClusterRestartIT.java

```diff
@@ -98,15 +76,21 @@ private void assertThatMlConfigIndexDoesNotExist() {
     }
 
     private void createAnomalyDetectorJob(String jobId) throws IOException {
-        Detector.Builder detector = new Detector.Builder("metric", "responsetime");
-        AnalysisConfig.Builder analysisConfig = new AnalysisConfig.Builder(Collections.singletonList(detector.build()))
-            .setBucketSpan(TimeValue.timeValueMinutes(10));
-        Job.Builder job = new Job.Builder(jobId)
-            .setAnalysisConfig(analysisConfig)
-            .setDataDescription(new DataDescription.Builder());
+        String jobConfig =
```

Comment:
We want to remove the dependency on x-pack core in these rest tests, taking out these builders helps.

`MlMigrationFullClusterRestartIT` still uses the core builders so we can't break that dependency yet but that test will be removed from this branch soon.